octobot_tentacles_manager/exporters/tentacle_package_exporter.py

```diff
@@ -62,5 +63,16 @@ async def prepare_export(self):
             self.copy_directory_content_to_working_dir(self.tentacles_folder,
                                                        ignore=self.tentacles_filter.should_ignore)
 
+    @staticmethod
+    def include_output_dir_in_package_name_if_any(artifact: models.TentaclePackage, output_dir: str) -> str:
+        """
+        Include artifact output_path in output_dir if any
+        :return: final output_dir
+        """
+        if artifact.output_path is not None and artifact.output_path:
```

Comment:
why not `if artifact.output_path` ?

Reply:
Yes it should be `if artifact.output_path` 👍